### PR TITLE
ResNet152 ONNX model inference on MXNet with EIA

### PR DESCRIPTION
**Pull request by Roshrini**
Adding example for showing resnet152 model from ONNX model zoo to run inference on EIA

### CODE REVIEW

**Comment by Roshrini** [comment]
@vandanavk

File: sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
...L59my7OOekPee87Hyd2LYvX7zCfnTItJ6KHLktm9RLlCtrFglFZkfAEJM+1xYhMMg9snRGzjtGaoIYk41Dagi3Qq2uqLUMTSWYM5YBtsySd3s/tALnkWouaxHSRibHU5YDl0RHL402m2xOO6xnf9sKH4HVpa9Sp5Yl3Ps7nP/Yprj1ykTe++lX60yX1YYPpFYcPDtiajGkPZ/iFxttEM+9YzmsGtuSN117jxpUr7N+/R1k6+vkSIoyw7E6njK5GTm8nIiMuXDa8Z/sSf/FPfzP18Sfpes1e/AWeeeZ93H91yWc/8Qq/9LGb/MSP/QI/83/9Dn7vBzXfe/ge/v6n73AYJyzmlhiXbKoCPw88OHhRZh2FpWRACgZlCvaHBm0diQJTjOh9ZG/5gOhbQX0YaZsIbVJmRmYwRBknoDqtYbCxbhVpbbBWoymJPnBxMpXIz3UR17PCcUTkFFq3Pa1uRd5pCzAK7RKVPpM0r4bDKSXqLpulEuw9eEjXdWxvbjIdjFgB/Vb/StuoJ/UelZERoffoLJFczRB9/t6aRhRh584FylgCQYyCuS2sokQOrtag1ff5rTy+IeSV4wvX0/O/5z8Dcus3V+arY0pKSchuMeAy8MxmIqVPHgK4wqx30fUwdj2Ik9XfOUdZlqSiQJuzPuHqoVS77q2tLswq+gzOFnmllGxCXQu6wCeLj5bGexbK0XQ9vRd9bOcjKvVUsZYFAsCIMazQ6axnafPJJIqhKQY5dgrcqKerGx7eu8dgWHL58kVOjhcEr8VJnLGsbwkbSXU2nOTfP/cdo84vnsT6Z5pqREgi8yqKitJVjKoBQ92swWNdlP5923frpCAVRfvbdx2+PcYYTW8sjc+29KDZU2AU6NBhY0NqFowKw2OTKe957p28511v40u/9lk++kv/lOPjQy5fvsiyntM2C5pmiVaJD3337+LCzgaVUzy8f1tyAmzJvG749Kc/SdsKmdM5y3S6wQvveYErly8ym53y+c99hr5p2dq9wNueeTt9gue/6QP83R//e9zfO+blL73OcGPEzXu3GG5ss2wjEQGTpXiWb5CSwumc9ANoKzxylCEYR6UCKZUSWH16k9e/9BmaxYloCYygnEMIOK0YbI1ZzGusjrhHpnzg930Xv/7GS1x+5iq1bjCV4+2PPcX9m3eZ+ILXX30F2p4JQ5gn6rbj5OiYS1u7L
```
Comment (lupesko) [inline sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb]: Remove

Comment (lupesko) [inline sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb]: Extra cell that is not necessary?

Comment (lupesko) [inline sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb]: 2019

**Comment by lupesko** [comment]
@domdivakaruni FYI

**Comment by Roshrini** [comment]
@lupesko Thanks for reviewing. Addressed all the comments.

**Comment by Roshrini** [comment]
@laurenyu Can you please review/merge this PR?

File: sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+   "source": [
+    "## Creating an inference endpoint and attaching an Elastic Inference(EI) accelerator\n",
+    "\n",
+    "Now we can use our `MXNetModel` object to build and deploy an `MXNetPredictor`. This creates a SageMaker Model and Endpoint, the latter of which we can use for performing inference. \n",
+    "\n",
+    "We pass the following arguments to the deploy method:\n",
+    "\n",
+    "* instance_count - how many instances to back the endpoint.\n",
+    "* instance_type - which EC2 instance type to use for the endpoint. For information on supported instance, please check here.\n",
+    "* accelerator_type - determines which EI accelerator type to attach to each of our instances. The supported types of accelerators can be found here: https://aws.amazon.com/sagemaker/pricing/instance-types/\n",
```
Comment (laurenyu) [inline sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb]: it's nice to put args, method names, etc. in backticks to make them monospace. I think there are a handful of instances throughout the notebook

File: sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hosting ONNX models with Amazon Elastic Inference\n",
+    "\n",
+    "The [Open Neural Network Exchange](https://onnx.ai/) (ONNX) is an open format for representing deep learning models with its extensible computation graph model and definitions of built-in operators and standard data types.\n",
+    "\n",
+    "In this example, we will use the ResNet-152v1 model from [Deep residual learning for image recognition](https://arxiv.org/abs/1512.03385). This model, alongside many others, can be found at the [ONNX Model Zoo](https://github.com/onnx/models).\n",
+    "\n",
```
[the remark this answers is not in the document]
Comment (Roshrini) [reply sagemaker-python-sdk/mxnet_onnx_eia/mxnet_onnx_eia.ipynb]: Done

**Comment by Roshrini** [comment]
@laurenyu Thanks for reviewing. Addressed all comments. Can you take a look again?

**Pull request merged**